recipe/meta.yaml

```diff
@@ -1,25 +1,22 @@
 # Full credit goes to https://github.com/conda/conda-recipes for providing this recipe.
 # It has been subsequently adapted for automated building with conda-forge.
 
-{% set version = "2.2.3" %}
+{% set version = "3.0.0" %}
 
 package:
   name: matplotlib
   version: {{ version }}
 
 source:
   url: https://github.com/matplotlib/matplotlib/archive/v{{ version }}.tar.gz
-  sha256: da5b804222864a8e854ed68f16dcbc8b2fa096537d84f879cc8289db368735c8
+  sha256: 4ab565581e3012fec06f93fc181d719a6f214f34c5b2a9f808741f0097507ea4
   patches:
     # Find libpng on Windows.
     - setupext.patch  # [win]
-    # A patch to make Qt the default backend.
-    - cfg_qt5agg.patch  # [not osx]
```

Comment:
@tacaswell I understand that there are some big changes in how we select the backend, right?
Is there anything we have to do here to ensure `qt` is selected or we no longer need that?

PS: I removed the patches from your branch to reflect the removal here.

Reply:
If PyQt is installed it will be selected (unless you are on a headless server in which case we pick Agg).

Reply:
Nice! What happens on OS X with `Qt` and Cocoa? Or when someone have `wx` and `qt` on Linux? (Sorry I should probably read the docs instead of asking here :grimacing:)

Reply:
The preference order for backends is MacOSX, Qt5Agg, Qt4Agg, GTK3Agg, GTK3Cairo, TkAgg, WXAgg, Agg, Cairo. It’s not in the documentation, only in the code here: https://github.com/matplotlib/matplotlib/blob/51c77d6ec01f4e8ca0ca947de8d3327d802678b5/lib/matplotlib/pyplot.py#L193

Reply:
Thanks for the explanation @marcelm!